flink-core/src/test/java/org/apache/flink/metrics/reporter/JMXReporterTest.java

```diff
@@ -88,7 +88,7 @@ public void testPortConflictHandling() throws Exception {
 		JMXReporter rep2 = new JMXReporter();
 
 		Configuration cfg1 = new Configuration();
-		cfg1.setString(ConfigConstants.METRICS_REPORTER_ARGUMENTS, "-port 9020-9035");
+		cfg1.setString("port", "9020-9035");
```

Comment:
The change of only starting a server when the ports are present should not change the config keys, correct?